docs/products/kafka/concepts/kafka-rest-authorization.rst

```diff
@@ -0,0 +1,3 @@
+Apache Kafka® REST proxy authorization
+======================================
+The Apache Kafka® REST proxy authorization, when :doc:`enabled <../karapace/howto/enable-karapace>` on your Aiven for Apache Kafka® service, allows the delegation of user authentication and authorization to Kafka. Apache Kafka® REST proxy authorization uses the authorization rules defined in :doc:`Apache Kafka® ACLs </docs/products/kafka/concepts/acl>`. 
```

Comment:
Why do we have the same content twice?

Reply:
Can we have an example to make it clear?

We can say that without auth you get access to all topics in read/write mode, while then with auth in, your REST access will follow the same rules as per ACLs